### DISCUSSION TITLE
Pitching without any glitches

### DISCUSSION THREAD

**Discussion by asd20202** [HomeOfAviSynthPlusEvolution/L-SMASH-Works#34] [Q&A]
I'm searching for a way to pitch audio with preserving the tempo without any glitches for a long time. 

Testing many tools, but I get weird sounding files or which are quite good, but not perfect.
Using Behappy is the best choice I think till today.
When you have a piano playing very slowly, you hear the glitches very well.
Its diffcult to say, how it sounds.
So my question is, can L-SMASH do pitch correction without increase time?

Thank you

**Comment by Asd-g** [comment]
No, L-SMASH cannot change the pitch at all (or the tempo). It's just decodes the audio. After that you can use Avisynth filter to do required filtering.
I guess you tried sox, if not - you can check it.
